### PR TITLE
Filter managed

### PR DESCRIPTION
**Pull request by haydenf**

### CODE REVIEW

File: src/components/DownloadContactListButton.tsx [hunk 1]
```diff
@@ -7,19 +7,27 @@ type DownloadButtonProps = {
 export function DownloadContactListButton (props: DownloadButtonProps) {
     let downloadUrl = null
     let csvData = '"Property Address","Property Type (Marketing)","Owner","Owner Mobile","Owner Phone","Contact","Contact Mobile","Contact Work Phone"\r\n'
+    const objectArr = props.results
```
Comment (johnEthicalTechnology) [inline src/components/DownloadContactListButton.tsx:10]: What's a name that gives more context and is more specific?

File: src/components/DownloadContactListButton.tsx [hunk 2]
```diff
@@ -7,19 +7,27 @@ type DownloadButtonProps = {
...
+    function getUniqueListBy (arr: UnprocessedResultsFromCRM[], key: string) {
+        return [...new Map(arr.map((propertyObject: any) => [propertyObject[key], propertyObject])).values()]
+    }
+    const filteredPropObject = getUniqueListBy(objectArr, 'id')
```
Comment (johnEthicalTechnology) [inline src/components/DownloadContactListButton.tsx:14]: More context and more specific variable name.

File: src/vendor/ZSDK.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/* eslint-disable */
+/* eslint-disable */ // remove this when building
```
Comment (johnEthicalTechnology) [inline src/vendor/ZSDK.js:1]: Did you have trouble with this as well?

**Review by johnEthicalTechnology — COMMENTED** [review]
There's probably going to be some conflicts when this is merged after the 1st PR. But that's okay. The only change that needs to be kept from the 1st PR is the addition of 'None' to the Managed array.

File: src/vendor/ZSDK.js (hunk 3 above)
Comment (haydenf) [reply src/vendor/ZSDK.js:1]: I'm not sure, I never built because I used the local. This was based off your advice

**Pull request merged**